cpp/ql/lib/semmle/code/cpp/dataflow/internal/DataFlow.qll

```diff
@@ -265,6 +272,10 @@ deprecated module Make<ConfigSig Config> implements GlobalFlowSig {
 module GlobalWithState<StateConfigSig Config> implements GlobalFlowSig {
   private module C implements FullStateConfigSig {
     import Config
+
+    predicate isSinkWithState(Node sink, FlowState state) { Config::isSink(sink, state) }
+
+    predicate isSinkWithAnyState(Node sink) { Config::isSink(sink) }
```

Comment:
This is not needed once we drop the renaming between the two signatures.

Reply:
Indeed - in 9a0a4a3611e7001624908f3dd1044fa3b4febb60 these predicates are just `isSink` now. I hope that was what you had in mind?